### PR TITLE
blinkit first page

### PR DESCRIPTION
**Pull request by amangupta3265**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="topNavbar">
+      <div class="delivery">
+        <div class="left">
+          <!-- <div class="logo"> -->
```
Comment (dk1710) [inline index.html]: remove the commented code

**Comment by amangupta3265** [comment]
image
image

https://user-images.githubusercontent.com/123450466/217489107-4b592bde-c16a-4856-bf75-c9ee3fc4ace5.mp4

File: index.html [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+      integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="topNavbar">
+      <div class="delivery">
```
Comment (dk1710) [inline index.html]: Check if we can do this with one div

File: index.html [hunk 3]
```diff
@@ -0,0 +1,449 @@
...
+        <div class="left">
+          <!-- <div class="logo"> -->
+          <ul>
```
Comment (dk1710) [inline index.html]: Check if list is required or not

File: index.html [hunk 4]
```diff
@@ -0,0 +1,449 @@
...
+                <span class="blink">blink</span><span class="it">it</span>
+              </h1>
+            </li>
+            <!-- </div> -->
+            <li>
+              <div>
+                <h2 class="deliveryTime">Delivery in 10 minutes</h2>
+                <form>
+                  <select>
+                    <option value="">Banglore,India</option>
```
Comment (dk1710) [inline index.html]: Add more options and check if select is working or not

File: index.html [hunk 5]
```diff
@@ -0,0 +1,449 @@
...
+            </li>
+
+            <!-- <ul>
+                        <li>Delhi,India</li>
+                        
+                    </ul> -->
+          </ul>
+        </div>
+        <div class="right">
+          <div class="login">
```
Comment (dk1710) [inline index.html]: Check if this extra div is required or not

File: index.html [hunk 6]
```diff
@@ -0,0 +1,449 @@
...
+            <button>Login</button>
+          </div>
+          <div class="myCart">
+            <button><i class="fa-solid fa-cart-shopping"></i>My Cart</button>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="navbar">
+      <hr class="lineColor" />
```
Comment (dk1710) [inline index.html]: This horizontal line should not be responsibility of 'navbar' div

File: index.html [hunk 7]
```diff
@@ -0,0 +1,449 @@
...
+        <li><a>Cold Drinks & Juices</a></li>
+        <li><a>Instant & Frozen Food</a></li>
+        <li><a>Tea, Coffee & Health Drinks</a></li>
+        <li><a>Bakery & Biscuits</a></li>
+        <li><a>More</a></li>
+      </ul>
+      <hr class="lineColor" />
+    </div>
+
+    <div class="container">
```
Comment (dk1710) [inline index.html]: rename container

File: index.html [hunk 8]
```diff
@@ -0,0 +1,449 @@
...
+      <div class="categoriesNav">
+        <ul>
+          <li>
+            <button id="defalutCategoriesOption">
```
Comment (dk1710) [inline index.html]: 1. Button is not required
2. For active list item we can use class also

File: index.html [hunk 9]
```diff
@@ -0,0 +1,449 @@
...
+            <form class="buyFilter">
+              <label for="sortBy" class="sortByLabel">Sort By</label>
+              <select id="sortBy" name="sortBy">
+                <option value="relevence">relevance</option>
+              </select>
+            </form>
+          </div>
+        </div>
+        <div class="items">
+          <ul>
```
Comment (dk1710) [inline index.html]: Check what is the better way than list

File: index.html [hunk 10]
```diff
@@ -0,0 +1,398 @@
...
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+      integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="topNavbar delivery">
```
Comment (dk1710) [inline index.html]: Why 2 classes needed

File: index.html [hunk 11]
```diff
@@ -0,0 +1,398 @@
...
+      <div class="left">
+        <h1 class="logo">
+          <span class="blink">blink</span><span class="it">it</span>
+        </h1>
+        <hr />
```
Comment (dk1710) [inline index.html]: no need of `hr`

File: index.html [hunk 12]
```diff
@@ -0,0 +1,398 @@
...
+
+        <div class="deliveryHead">
+          <h2 class="deliveryTime">Delivery in 10 minutes</h2>
```
Comment (dk1710) [inline index.html]: Let's decrease the size of this

File: index.html [hunk 13]
```diff
@@ -0,0 +1,398 @@
...
+      </div>
+      <div class="right">
+        <button type="submit" class="login">Login</button>
+        <button class="myCart">
+          <i class="fa-solid fa-cart-shopping"></i>My Cart
+        </button>
+      </div>
+    </div>
+
+    <hr class="lineColor" />
```
Comment (dk1710) [inline index.html]: Let's do this by border-bottom

File: index.html [hunk 14]
```diff
@@ -0,0 +1,398 @@
...
+  </head>
+  <body>
+    <div class="topNavbar delivery">
+      <div class="left">
+        <h1 class="logo">
+          <span class="blink">blink</span><span class="it">it</span>
+        </h1>
+        <hr />
+
+        <div class="deliveryHead">
```
Comment (dk1710) [inline index.html]: deliveryHead <> delivery

File: index.html [hunk 15]
```diff
@@ -0,0 +1,398 @@
...
+        <li><a>Instant & Frozen Food</a></li>
+        <li><a>Tea, Coffee & Health Drinks</a></li>
+        <li><a>Bakery & Biscuits</a></li>
+        <li><a>More</a></li>
+      </ul>
+    </div>
+
+    <hr class="lineColor" />
+
+    <div class="containerForBuyingItems">
```
Comment (dk1710) [inline index.html]: containerForBuyingItems <> itemsContainer or productsContainer

File: index.html [hunk 16]
```diff
@@ -0,0 +1,398 @@
...
+      <div class="categoriesNav">
```
Comment (dk1710) [inline index.html]: categoriesNav <> productCategoriesNavbar

File: index.html [hunk 17]
```diff
@@ -0,0 +1,398 @@
...
+        <ul>
+          <li class="active">
+            <div class="categoriesNavButton">
```
Comment (dk1710) [inline index.html]: 1. categoriesNavButton <> productCategory
2. Check if this div is required or not

File: index.html [hunk 18]
```diff
@@ -0,0 +1,398 @@
...
+              <img class="categoriesNavImage" src="./potato.jpeg" />
+              <div>
```
Comment (dk1710) [inline index.html]: Check if this div is required or not

File: index.html [hunk 19]
```diff
@@ -0,0 +1,398 @@
...
+              <img class="categoriesNavImage" src="./images/freshly cut.webp" />
+              <div>
+                <span class="categoriesNavName">Freshly Cut & Sprouts</span>
+              </div>
+            </div>
+          </li>
+        </ul>
+      </div>
+      <div class="itemsSection">
+        <div class="itemsHead">
```
Comment (dk1710) [inline index.html]: itemsHead <> itemsHeading

File: index.html [hunk 20]
```diff
@@ -0,0 +1,398 @@
...
+          <div class="buyHeading">
```
Comment (dk1710) [inline index.html]: Check if this div is required or not

File: index.html [hunk 21]
```diff
@@ -0,0 +1,398 @@
...
+            </form>
+          </div>
+        </div>
+        <div class="items">
+          <div class="singleItem">
+            <div class="itemImage">
+              <div class="offer">10% OFF</div>
+              <img src="./potato.jpeg" />
+            </div>
+            <div class="sourced">sourced at : 5am</div>
```
Comment (dk1710) [inline index.html]: let's move this div to `itemImage`

File: index.html [hunk 22]
```diff
@@ -0,0 +1,398 @@
...
+            <div class="itemName">Potato(Aloo)</div>
+            <div class="itemWeight">1Kg</div>
+            <div class="itemPriceQuantity">
+              <div class="itemPrice">
+                <div>
```
Comment (dk1710) [inline index.html]: extra div

File: index.html [hunk 23]
```diff
@@ -0,0 +1,398 @@
...
+        <div class="items">
+          <div class="singleItem">
+            <div class="itemImage">
+              <div class="offer">10% OFF</div>
+              <img src="./potato.jpeg" />
+            </div>
+            <div class="sourced">sourced at : 5am</div>
+            <div class="itemName">Potato(Aloo)</div>
+            <div class="itemWeight">1Kg</div>
+            <div class="itemPriceQuantity">
```
Comment (dk1710) [inline index.html]: itemPriceQuantity <> itemDetails

File: index.html [hunk 24]
```diff
@@ -0,0 +1,398 @@
...
+              <div class="itemPrice">
+                <div>
+                  <span class="newPrice"><span>&#8377;</span>15</span>
+                  <span class="oldPrice"><span>&#8377;</span>15</span>
+                </div>
+              </div>
+              <div class="itemQuantity">
```
Comment (dk1710) [inline index.html]: itemQuantity <>selectedItemQuantity

File: index.html [hunk 25]
```diff
@@ -0,0 +1,398 @@
...
+                <button>ADD</button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="advantages">
+      <div class="adanvatgesDesciption">
```
Comment (dk1710) [inline index.html]: adanvatgesDesciption <> adanvatgeCard

File: index.html [hunk 26]
```diff
@@ -0,0 +1,398 @@
...
+      “Blinkit” is owned & managed by "Blink Commerce Private Limited" (formerly
+      known as Grofers India Private Limited) and is not related, linked or
+      interconnected in whatsoever manner or nature, to “GROFFR.COM” which is a
+      real estate services business operated by “Redstone Consultancy Services
+      Private Limited”.
+    </div>
+
+    <hr class="footerLine" />
+    <div class="footerColor">
+      <div class="footer">
```
Comment (dk1710) [inline index.html]: extra div

File: index.html [hunk 27]
```diff
@@ -0,0 +1,398 @@
...
+        <div class="companyName">
+          © Blink Commerce Private Limited (formerly known as Grofers India
+          Private Limited), 2016-2023
+        </div>
+        <div class="downloadApp">
+          <div class="downloadAppTitle">Download App</div>
+          <div class="appStore">
```
Comment (dk1710) [inline index.html]: Check if this div is required or not

File: index.html [hunk 28]
```diff
@@ -0,0 +1,398 @@
...
+            <img src="./images/appstore.webp" />
+          </div>
+          <div class="playStore">
```
Comment (dk1710) [inline index.html]: Check if this div is required or not

File: index.html [hunk 29]
```diff
@@ -0,0 +1,388 @@
...
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+      integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="delivery">
```
Comment (dk1710) [inline index.html]: rename delivery

File: index.css [hunk 30]
```diff
@@ -0,0 +1,289 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+.blink {
```
Comment (dk1710) [inline index.css]: blink <> greenColor

File: index.css [hunk 31]
```diff
@@ -0,0 +1,289 @@
...
+  color: #f8cb45;
+}
+.it {
```
Comment (dk1710) [inline index.css]: it <> yellowColor

File: index.css [hunk 32]
```diff
@@ -0,0 +1,289 @@
...
+  color: #54b226;
+}
+.delivery {
+  height: 60px;
+  display: flex;
+  flex-direction: row;
```
Comment (dk1710) [inline index.css:15]: `display: flex;
  flex-direction: row;`
You have repeated above code at many places. 
You can create separate class for just that which will be called `flex-row` and you can reuse that all places

File: index.css [hunk 33]
```diff
@@ -0,0 +1,289 @@
...
+}
+.left,
+.right {
+  display: flex;
+  flex-direction: row;
+}
+.delivery .logo {
+  justify-content: flex-start;
+  padding: 10px 25px;
+  font-weight: 700;
```
Comment (dk1710) [inline index.css]: You are already using `h1` tag then why are we using `font-weight` ?

File: index.css [hunk 34]
```diff
@@ -0,0 +1,289 @@
...
+}
+
+.right .login {
+  border: none;
+  color: gray;
+  background-color: #f5f5f5;
+  padding: 10px;
+  border-radius: 5px;
+}
+.delivery {
```
Comment (dk1710) [inline index.css]: Style for this class is already defined above

File: index.css [hunk 35]
```diff
@@ -0,0 +1,289 @@
...
+  margin-right: 30px;
+  color: #fff;
+  background-color: #23963f;
+  border: none;
+  border-radius: 5px;
+  margin-left: 30px;
+}
+.adanvatgesDesciption h3,
+.delivery .deliveryHead,
+.delivery .right {
```
Comment (dk1710) [inline index.css]: Create separate class for this styles and give some other name and reuse that at all 3 places

File: index.css [hunk 36]
```diff
@@ -0,0 +1,287 @@
...
+}
+
+.adanvatgeCard h3,
+.delivery .deliveryHead,
+.delivery .right {
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.right .login {
```
Comment (dk1710) [inline index.css]: 1. Why are you accessing `right` here? Can't you directly use `login` ?
2. login <> loginBtn

File: index.css [hunk 37]
```diff
@@ -0,0 +1,287 @@
+* {
```
Comment (dk1710) [inline index.css:1]: change all the pixel sizes to `rem`

File: index.html [hunk 38]
```diff
@@ -0,0 +1,388 @@
...
+      </div>
+      <div class="right">
+        <button type="submit" class="login">Login</button>
+        <button class="myCart">
+          <i class="fa-solid fa-cart-shopping"></i>My Cart
+        </button>
+      </div>
+    </div>
+
+    <div class="navbar">
```
Comment (dk1710) [inline index.html]: Check if we can remove this div

File: index.html [hunk 39]
```diff
@@ -0,0 +1,388 @@
...
+        <li><a>Bakery & Biscuits</a></li>
+        <li><a>More</a></li>
+      </ul>
+    </div>
+
+    <div class="productsContainer">
+      <div class="categoriesNav">
+        <ul>
+          <li class="active">
+            <div class="productCategories">
```
Comment (dk1710) [inline index.html]: 1. productCategories <> productCategory
2. Why is this div required?

File: index.html [hunk 40]
```diff
@@ -0,0 +1,387 @@
...
+      <div class="right flexRow paddingTopBottom10px">
+        <button type="submit" class="loginBtn">Login</button>
+        <button class="myCart">
+          <i class="fa-solid fa-cart-shopping"></i>My Cart
+        </button>
+      </div>
+    </div>
+
+
+    <ul class=" navbar categories">
```
Comment (dk1710) [inline index.html]: Why two classes needed?

File: index.html [hunk 41]
```diff
@@ -0,0 +1,387 @@
...
+          <select>
+            <option value="">Banglore,India</option>
+            <option value="">Delhi,India</option>
+            <option value="">Mumbai,India</option>
+            <option value="">Kolkata,India</option>
+          </select>
+        </div>
+      </div>
+      <div class="right flexRow paddingTopBottom10px">
+        <button type="submit" class="loginBtn">Login</button>
```
Comment (dk1710) [inline index.html]: Why `submit` ?

File: index.html [hunk 42]
```diff
@@ -0,0 +1,387 @@
...
+      <div class="adanvatgeCard">
+        <img src="./images/fast dilevery.webp" />
+        <h3 class="paddingTopBottom10px">Superfast Delivery</h3>
+        <p>
+          Get your order delivered to your doorstep at the earliest from dark
+          stores near you
+        </p>
+      </div>
+      <div class="adanvatgeCard">
+        <img src=".//images/deliver-icon-earnings.webp" />
```
Comment (dk1710) [inline index.html]: check the path

File: index.css [hunk 43]
```diff
@@ -0,0 +1,280 @@
...
+.loginBtn:hover {
+  background-color: #f5f5f5;
+}
+
+.categories {
+  color: gray;
+  border-bottom: 0.0625rem solid #f5f5f5;
+}
+.categories li:first-child {
+  margin-left: 13%;
```
Comment (dk1710) [inline index.css]: It shouldn't be needed, try different approach

File: index.css [hunk 44]
```diff
@@ -0,0 +1,280 @@
...
+.categories li {
+  display: inline-block;
+  padding: 0.5rem;
+}
+.categories li:hover {
+  background-color: #f5f2f2;
+  cursor: pointer;
+}
+.productsContainer {
+  margin-left: 15%;
```
Comment (dk1710) [inline index.css]: check different property of `margin`

File: index.css [hunk 45]
```diff
@@ -0,0 +1,276 @@
...
+}
+.logo__yellowColor {
+  color: #f8cb45;
+}
+.logo__greenColor {
+  color: #54b226;
+}
+.flexRow {
+  display: flex;
+  flex-direction: row;
```
Comment (dk1710) [inline index.css]: 1. Default value of `flex-direction` is `row`, so we don't need to give the same value
2. We can add `align-items: center`, because you have used that many places

File: index.css [hunk 46]
```diff
@@ -0,0 +1,276 @@
...
+  height: 3.75rem;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 0.0625rem solid #f5f5f5;
+}
+.deliveryHead {
+  margin-left: 3.125rem;
+}
+.topNavbar .logo {
+  justify-content: flex-start;
```
Comment (dk1710) [inline index.css]: Why is this required?

File: index.css [hunk 47]
```diff
@@ -0,0 +1,276 @@
...
+  margin-right: 0.625rem;
+}
+.paddingTopBottom10px {
+  padding-top: 0.625rem;
+  padding-bottom: 0.625rem;
+}
+.categories {
+  color: gray;
+  border-bottom: 0.0625rem solid #f5f5f5;
+  display: flex;
```
Comment (dk1710) [inline index.css]: use `flexRow` class

File: index.css [hunk 48]
```diff
@@ -0,0 +1,276 @@
...
+  justify-content: center;
+}
+.categories li {
+  display: inline-block;
```
Comment (dk1710) [inline index.css]: Why is this required?

File: index.html [hunk 49]
```diff
@@ -0,0 +1,343 @@
...
+      <li>Tea, Coffee & Health Drinks</li>
+      <li>Bakery & Biscuits</li>
+      <li>More</li>
+    </ul>
+
+    <div class="productsContainer">
+      <div class="categoriesNav">
+        <ul>
+          <li class="productCategory active">
+            <img class="productCategory__Image" src="./potato.jpeg" />
```
Comment (dk1710) [inline index.html]: `productCategory__Image` <> `productCategory__image`

File: index.html [hunk 50]
```diff
@@ -0,0 +1,343 @@
...
+
+            <span class="productCategory__Name">All</span>
```
Comment (dk1710) [inline index.html]: `productCategory__Name` <> `productCategory__name`

File: index.html (hunk 16 above)
Comment (dk1710) [reply index.html]: Please resolve this comment

File: index.css [hunk 51]
```diff
@@ -0,0 +1,276 @@
...
+}
+.categories li:hover {
+  background-color: #f5f2f2;
+  cursor: pointer;
+}
+.productsContainer {
+  margin: auto;
+  width: 70%;
+  min-height: 37.5rem;
+  display: flex;
```
Comment (dk1710) [inline index.css]: use flexRow class

File: index.css [hunk 52]
```diff
@@ -0,0 +1,276 @@
...
+  padding: 0.5rem;
+}
+.categories li:hover {
+  background-color: #f5f2f2;
+  cursor: pointer;
+}
+.productsContainer {
+  margin: auto;
+  width: 70%;
+  min-height: 37.5rem;
```
Comment (dk1710) [inline index.css]: need to discuss this

File: index.html [hunk 53]
```diff
@@ -0,0 +1,343 @@
...
+      <li>Instant & Frozen Food</li>
+      <li>Tea, Coffee & Health Drinks</li>
+      <li>Bakery & Biscuits</li>
+      <li>More</li>
+    </ul>
+
+    <div class="productsContainer">
+      <div class="categoriesNav">
+        <ul>
+          <li class="productCategory active">
```
Comment (dk1710) [inline index.html]: `productCategory active` <> `productCategory__active`

File: index.css [hunk 54]
```diff
@@ -0,0 +1,276 @@
...
+}
+.productCategory__Image {
+  width: 3.125rem;
+  height: 3.125rem;
+  float: left;
+  margin-right: 0.625rem;
+  border-radius: 0.3125rem;
+}
+.productCategory {
+  display: flex;
```
Comment (dk1710) [inline index.css]: use flexRow class

File: index.css [hunk 55]
```diff
@@ -0,0 +1,276 @@
...
+}
+.categoriesNav li {
+  max-width: 15rem;
+  padding: 0.5rem;
+  background-color: #fff;
+  border: 0.0625rem solid #f5f5f5;
+}
+.categoriesNav .active,
+.categoriesNav li:hover {
+  border-left: 0.1875rem solid #23963f;
```
Comment (dk1710) [inline index.css]: While hovering we don't have to show border left

File: index.css [hunk 56]
```diff
@@ -0,0 +1,276 @@
...
+  background-color: #ebfeef;
+}
+.productCategory__Image {
+  width: 3.125rem;
+  height: 3.125rem;
+  float: left;
```
Comment (dk1710) [inline index.css]: why is this required?

File: index.css [hunk 57]
```diff
@@ -0,0 +1,276 @@
...
+  align-items: center;
+  justify-content: flex-start;
+  flex-direction: row;
+  padding: 0 0.3125rem;
+}
+.productCategory__Name {
+  font-weight: 500;
+}
+.productsSection {
+  width: 80%;
```
Comment (dk1710) [inline index.css:134]: need to discuss this

File: index.html [hunk 58]
```diff
@@ -0,0 +1,343 @@
...
+          <form class="buyFilter">
+            <label for="sortBy" class="sortByLabel">Sort By</label>
+            <select id="sortBy" name="sortBy">
+              <option value="relevence">Relevance</option>
+            </select>
+          </form>
+        </div>
+        <div class="products">
+          <div class="product">
+            <div class="product__Image">
```
Comment (dk1710) [inline index.html]: `product__Image` <> `product__image`
fix this change at other places also

File: index.css [hunk 59]
```diff
@@ -0,0 +1,276 @@
...
+  min-height: 43.75rem;
+}
+.productsHeading {
+  display: flex;
```
Comment (dk1710) [inline index.css:140]: use flexRow class

File: index.html [hunk 60]
```diff
@@ -0,0 +1,343 @@
...
+              class="productCategory__Image"
+              src="./images/freshly cut.webp"
+            />
+
+            <span class="productCategory__Name">Freshly Cut & Sprouts</span>
+          </li>
+        </ul>
+      </div>
+      <div class="productsSection">
+        <div class="productsHeading">
```
Comment (dk1710) [inline index.html]: `productsHeading` <> `productsSection__heading`

File: index.html [hunk 61]
```diff
@@ -0,0 +1,343 @@
...
+          <h4 class="buyHeading">Buy Fresh Vegetables Online</h4>
+
+          <form class="buyFilter">
```
Comment (dk1710) [inline index.html]: `buyFilter ` <> `productsSection__filter`

File: index.html [hunk 62]
```diff
@@ -0,0 +1,343 @@
...
+            <label for="sortBy" class="sortByLabel">Sort By</label>
+            <select id="sortBy" name="sortBy">
```
Comment (dk1710) [inline index.html]: Why are we giving `id` ?

File: index.html [hunk 63]
```diff
@@ -0,0 +1,343 @@
...
+              <option value="relevence">Relevance</option>
+            </select>
+          </form>
+        </div>
+        <div class="products">
+          <div class="product">
```
Comment (dk1710) [inline index.html]: use BEM for classname

File: index.css [hunk 64]
```diff
@@ -0,0 +1,276 @@
...
+  font-size: 0.8rem;
+}
+#sortBy {
+  appearance: none;
+  padding: 0.1875rem;
+  color: #23963f;
+  width: 10rem;
+}
+.products {
+  display: flex;
```
Comment (dk1710) [inline index.css]: use flexRow class

File: index.css [hunk 65]
```diff
@@ -0,0 +1,276 @@
...
+  flex-wrap: wrap;
+}
+.products .product {
+  width: 11.21875rem;
+  padding: 0.625rem;
+  border: 0.0625rem solid #f5f5f5;
+}
+.product {
```
Comment (dk1710) [inline index.css]: this classname style is already defined above

File: index.css [hunk 66]
```diff
@@ -0,0 +1,276 @@
...
+  position: absolute;
+  top: 0.3125rem;
+  left: 0.3125rem;
+}
+.product__Details {
+  display: flex;
+  justify-content: space-between;
+}
+.product__Name,
+.product__Weight {
```
Comment (dk1710) [inline index.css]: this margin is little bit different, please check in design

File: index.css [hunk 67]
```diff
@@ -0,0 +1,276 @@
...
+  color: gray;
+}
+.product__Quantity button {
+  padding: 0.625rem 1.5625rem;
+  border-radius: 0.3125rem;
+  border: 0.0625rem solid #23963f;
+  font-weight: 600;
+}
+.advantages {
+  display: flex;
```
Comment (dk1710) [inline index.css]: use `flexRow` class

File: index.css [hunk 68]
```diff
@@ -0,0 +1,276 @@
...
+  justify-content: space-evenly;
+  align-items: center;
+}
+.adanvatgeCard {
+  width: 20%;
+  text-align: center;
+  border-radius: 0.3125rem;
+  border: 0.125rem solid #f5f5f5;
+  margin-right: -30%;
+  margin-top: 1.875rem;
```
Comment (dk1710) [inline index.css]: Why are we giving `margin-top` to card and not container?

File: index.css [hunk 69]
```diff
@@ -0,0 +1,276 @@
...
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+.adanvatgeCard {
+  width: 20%;
+  text-align: center;
+  border-radius: 0.3125rem;
+  border: 0.125rem solid #f5f5f5;
+  margin-right: -30%;
```
Comment (dk1710) [inline index.css]: Why are we giving margin in percentage?

File: index.html [hunk 70]
```diff
@@ -0,0 +1,343 @@
...
+                <button>ADD</button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="advantages">
+      <div class="adanvatgeCard">
```
Comment (dk1710) [inline index.html]: use BEM

File: index.html [hunk 71]
```diff
@@ -0,0 +1,343 @@
...
+      “Blinkit” is owned & managed by "Blink Commerce Private Limited" (formerly
+      known as Grofers India Private Limited) and is not related, linked or
+      interconnected in whatsoever manner or nature, to “GROFFR.COM” which is a
+      real estate services business operated by “Redstone Consultancy Services
+      Private Limited”.
+    </div>
+
+    <hr class="footerLine" />
+    <div class="footer">
+      <div class="companyName">
```
Comment (dk1710) [inline index.html]: use BEM, below as well

File: index.css [hunk 72]
```diff
@@ -0,0 +1,276 @@
...
+}
+.adanvatgeCard img {
+  width: 6.25rem;
+  height: 6.25rem;
+}
+.adanvatgeCard p {
+  color: gray;
+}
+.aboutBlinkit {
+  margin: 2.5rem 8% 2.5rem 12%;
```
Comment (dk1710) [inline index.css]: try different approach

File: index.css [hunk 73]
```diff
@@ -0,0 +1,276 @@
...
+  color: gray;
+}
+.footerLine {
+  width: 80%;
+  margin-left: 12%;
```
Comment (dk1710) [inline index.css]: try different approach

File: index.css [hunk 74]
```diff
@@ -0,0 +1,276 @@
...
+  margin-bottom: 1.875rem;
+  border: 0.0625rem solid #f5f5f5;
+}
+.downloadApp img {
+  width: 6.25rem;
+  height: 2.5rem;
+}
+.footer {
+  display: grid;
```
Comment (dk1710) [inline index.css]: no need for grid template, please check when should we grid template is useful.

File: index.css [hunk 75]
```diff
@@ -0,0 +1,276 @@
...
+  margin-right: 1.25rem;
+  width: 62.5rem;
+  display: inline;
+}
+.socialHandles {
+  margin-left: 12rem;
+}
+.socialHandles i {
+  font-size: 2.5rem;
+  padding-left: 1rem;
```
Comment (dk1710) [inline index.css]: What is the use case of this?
